From 7fcca9baefe9000b8806c8385cd8652245c846e4 Mon Sep 17 00:00:00 2001
From: Jan-Erik Rediger <badboy@archlinux.us>
Date: Wed, 13 Aug 2014 19:31:57 +0200
Subject: [PATCH 33/39] Don't use extended Regexp Syntax

It's not POSIX (BSD systems have -E instead) and we don't actually need it.

Closes #1922
---
 utils/install_server.sh | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/utils/install_server.sh b/utils/install_server.sh
index 98e047e3..39fd9875 100755
--- a/utils/install_server.sh
+++ b/utils/install_server.sh
@@ -135,13 +135,13 @@ fi
 echo "## Generated by install_server.sh ##" > $TMP_FILE
 
 read -r SED_EXPR <<-EOF
-s#^port [0-9]{4}\$#port ${REDIS_PORT}#; \
-s#^logfile .+\$#logfile ${REDIS_LOG_FILE}#; \
-s#^dir .+\$#dir ${REDIS_DATA_DIR}#; \
-s#^pidfile .+\$#pidfile ${PIDFILE}#; \
-s#^daemonize no\$#daemonize yes#;
+s#^port .\+#port ${REDIS_PORT}#; \
+s#^logfile .\+#logfile ${REDIS_LOG_FILE}#; \
+s#^dir .\+#dir ${REDIS_DATA_DIR}#; \
+s#^pidfile .\+#pidfile ${PIDFILE}#; \
+s#^daemonize no#daemonize yes#;
 EOF
-sed -r "$SED_EXPR" $DEFAULT_CONFIG  >> $TMP_FILE
+sed "$SED_EXPR" $DEFAULT_CONFIG >> $TMP_FILE
 
 #cat $TPL_FILE | while read line; do eval "echo \"$line\"" >> $TMP_FILE; done
 cp $TMP_FILE $REDIS_CONFIG_FILE || die "Could not write redis config file $REDIS_CONFIG_FILE"
-- 
2.14.1

